src/common/database.c

```diff
@@ -1337,7 +1337,7 @@ void dt_database_show_error(const dt_database_t *db)
 {
   if(!db->lock_acquired)
   {
-    char *label_text = g_markup_printf_escaped(_("an error has occured while trying to open the database from\n"
+    char *label_text = g_markup_printf_escaped(_("an error has occurred while trying to open the database from\n"
```

Comment:
This is a translatable string. It may not be changed. Please move that line to another PR that can be merged next year.